dags/anomdtct.py

```diff
@@ -46,14 +46,3 @@
             wait_for_clients_first_seen,
         ]
     )
-
-    deviations = bigquery_etl_query(
-        task_id="deviations",
-        project_id="moz-fx-data-shared-prod",
-        destination_table="deviations_v1",
-        dataset_id="telemetry_derived",
-        arguments=("--replace",),
```

Comment:
Just noticed this. We need to add support for these parameters.

Reply:
Does this line have any effect? The bigqueryrc in the image sets `--replace=true` as the default.

Reply:
Oh right, then it doesn't make a difference here. There are some tasks that set other arguments though, so generally I think we want to support defining arguments.